src/components/HelpModal/BorrowInterestHelpModal/index.tsx

```diff
@@ -1,60 +1,14 @@
-import React from 'react';
-import { useIntl } from 'react-intl';
-
-import { useThemeContext } from '@pret/pret-ui-kit';
-import HelpItem from '../../HelpItem';
-import TextWithModal from '../../TextWithModal';
-
-import messages from './messages';
-
 import { HelpModalProps } from '../types';
 
-export default function BorrowInterestHelpModal({
-  text,
-  iconSize,
-  className,
-  lightWeight,
-  color,
-  onWhiteBackground,
-}: HelpModalProps) {
-  const intl = useIntl();
-  const { currentTheme } = useThemeContext();
-
+export default function BorrowInterestHelpModal({ text }: HelpModalProps) {
```

Comment:
Should we comment out instead of removing the code since the stable borrow feature may be enabled later?